gen/app-gen

```diff
@@ -34,7 +34,7 @@ Usage: $0 [options] [-- [options-for-mender-artifact] ]
         --output-path       - Path to output artifact file. Default: reboot-artifact.mender
         --image [current-url,]new-url       - Path to output artifact file. Default: reboot-artifact.mender
         --deep-delta        - Calculate delta by parsing the image
-        --manifests-dir     - Directory containing orchestrator-specific manifests describing the deployment
+        --manifests-dir [current-dir,]new-dir     - Directory containing orchestrator-specific manifests describing the deployment
```

Comment:
we should fix the description a bit, explaining what will happen when two paths are given.